### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LatiuAnaMaria**

### CODE REVIEW

**Comment by LatiuAnaMaria** [comment]
<!DOCTYPE html>
<html>
    <head>
        <title>Pixel Art Maker</title>
        <metacharset="utf-8">
        <lang="en">
        <link rel="stylesheet" href="PixelArtMaker.css">
        <link href="Project_Javascript_jQuery.css">
        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Monoton">
        <script id="ud-grader-options">UdacityFEGradingEngine.turnOn();</script>
    </head>
    <body>
        <header  id="header" class="header">
            <h1>Lab: Pixel Art Maker</h1>
        </header> 
        <nav id="navbar"></nav>       
        <main id="main" class="main">  
            <h2>Choose Grid Size</h2>
            <form id="sizePicker" class="sizePicker"> 
                <label>Grid Height:(1-18)</label>
                <input class="input" type="number"  id="heightGrid" min="1" max="18" value="1"></input>  
                <label id="secondLabel">Grid Width:(1-18)</label>
                <input type="number" class="input"  id="widthGrid" min="1" max="18" value="1"></input>
                <input onclick ="makeGrid()" type="submit" id="submitButton" class="newGrid" value="Submit" >
                <input id="reset" type="button" value="Reset" class="newGrid" >
            </form>
            <h2>Pick A Color</h2>
                <input type="color" id="colorPicker" class="colorPicker">
        </main>  
            <h2>Design Canvas</h2> 
            <div id="canvas">      
            <table id="pixelCanvas" class="pixelCanvas"></table>
        </div>
        </div>
        <footer id="footer"></footer>
        <script src="PixelArtMaker.js"></script>
    </body>
</html>

**Comment by LatiuAnaMaria** [comment]
html, body {      
   width: 100%vw;
   min-height: 100vh;
   text-align: center;  
   margin:0;
   padding:0;   
}
body {
     display:grid;   
    grid-template-rows: 150px auto 150px; 
    /*grid-template-columns: minmax(200px, 3fr) 9fr;*/
    background: linear-gradient(270deg, #23a6d5, #23d5ab);      
    text-align: center;  
    color:black;     
}
.header {        
    width: 100%;
    height: 90px;
    text-align: center;
    background-color:white;
    margin: 10 auto;
    grid-column: 1 / 3;
   }

h1 {
    font-weight: bold;
    /*font-family: monoton;*/
    font-family: Rockwell Extra Bold; 
    color:#0000e6;
    font-size: 70px;
 }

h2 {
    margin: 1em 0 0.25em;
    color:#0000e6;
}

h2:first-of-type {
    margin-top: 0.5em;
}

table,
tr,
td {
    border: 1px solid #0000e6;
}

table {
    border-collapse: collapse;
    margin: 0 auto;
}

input[type=number] {
    width: 6em;
}
.input{
    width:60px;
    height:13px;
    border:2px solid #0000e6;
    }

#colorPicker {
    width:50px;
    height:27px;

}
#sizePicker {
    color:#0000e6;
    margin-top:20px;
    margin-bottom: 20px;
}

.newGrid{
    width:60px;
    height:23px;
    border:2px solid #0000e6;    
    margin-left: 120px;    
    background-color:white;
     
    }
#secondLabel {
        margin-left: 100px;
    }
#pixelCanvas {
    width: 394px;
    height: 394px;
    border: solid #0000e6; 
    margin: 30px auto;  
}  
footer {
    grid-column: 1 / 3;
    width: 10%;
    height: 50px;
} 
#canvas {
    border:1px #0000e6;
}

**Comment by LatiuAnaMaria** [comment]
function makeGrid() {
    const rows = $('#heightGrid').val();
    const cols = $('#widthGrid').val(); 
    const table = $('#pixelCanvas');   
    // Removing old table if it exsists
    $('#pixelCanvas').children().remove();
    // Creating table rows
    for (let i = 0; i < rows; i++) {
        $('#pixelCanvas').append('<tr></tr>');
    }
    // Creating cells to every row
    for (let j = 0; j < cols; j++) {
            $('#pixelCanvas tr').append('<td></<td>');
        }
         /*let j = 0;
        while (j < cols) {
    $('#containerGrid tr').append('<td></<td>');
    j++;
}*/
        table.on("click", "td", function()  {
        var color= $("input[type='color']").val();
        $(this).attr("bgcolor",color);
    });
}
$("input[type='submit']").click(function(e) {
    e.preventDefault();
    makeGrid();
});
$('#reset').click(function() {
  $('#pixelCanvas').children().remove();  
});
